internal/db/rdb/mysql/mysql.go

```diff
@@ -141,6 +143,14 @@ func (m *mySQLClient) Open(ctx context.Context) (err error) {
 // Ping check the connection of MySQL database.
 // If the connection is closed, it returns error.
 func (m *mySQLClient) Ping(ctx context.Context) (err error) {
+	if m.session == nil {
+		err = errors.ErrMySQLSessionNil
+		log.Debugf(
+			"err: %s, { db: %s, host: %s, port: %d, user: %s, pass: %s, name: %s, charset: %s } ",
+			err.Error(), m.db, m.host, m.port, m.user, m.pass, m.name, m.charset,
```

Comment:
we should not print the password to the log

Reply:
@kevindiu 
why did you suggest as above?

Reply:
I think we can use `err` instead of `err.Error()` in `Debugf()`.
https://stackoverflow.com/questions/43887776/what-is-the-correct-format-specifier-to-print-error-object-in-go-s-or-v

Reply:
it is correct, but we use glg as a log package.

Reply:
The final output is the same, so I think it doesn't matter which one 🤔